eradiate/radprops/particle_dist.py

```diff
@@ -288,27 +220,33 @@ class Array(ParticleDistribution):
                 attr.validators.instance_of(xr.DataArray)
             ),
         ),
-        doc="Particle vertical distribution data array. Number fraction as a "
-        "function of altitude (``z``).",
-        type=":class:`xarray.DataArray`",
+        doc="Particle distribution data array. Number fraction as a "
+        "function of altitude (``z``).\n"
+        "Note that number fraction do not need to be normalised.",
+        type=":class:`~xarray.DataArray`",
         default="``None``",
     )
 
+    @data_array.validator
+    def _validate_data_array(instance, attribute, value):
```

Comment:
This function is defined twice, isn't it a problem?

Reply:
I assume it is, I renamed that function